src/components/WelcomeBack/UnlockWallet.vue

```diff
@@ -1,17 +1,20 @@
 <template>
   <div class="unlock-wallet">
     <p class="text">Insert a password to unlock wallet</p>
-    <input v-model="password" type="password">
-
-    <button @click="unlockWallet">
+    <input class="password-input" v-model="password" type="password" placeholder ="••••••••">
+    <div class="submit">
+    <Button :onClick="unlockWallet" type="primary">Unlock</Button>
+    <!-- <button class="unlock-btn" @click="unlockWallet">
       Unlock
-    </button>
+    </button> -->
+    </div>
     <p v-if="showError">Invalid password</p>
   </div>
 </template>
 
 <script>
 import { mapState } from 'vuex'
+import Button from '../Button'
```

Comment:
Instead of use relative imports, you can use `@` that is a binding to `./src`
```suggestion
import Button from '@/components/Button'
```